docs/kyma-environment-broker/03-01-service-description.md

```diff
@@ -208,3 +209,26 @@ These are the provisioning parameters for the `own_cluster` plan that you config
 | **shootDomain** | string | Domain of the shoot where you install Kyma. | Yes | None |
 | **shootName** | string | Name of the shoot where you install Kyma. | Yes | None |
 
+## Preview cluster plan
```

Comment:
I dont know why, but when previewing this file I don't see this section see: https://github.com/kyma-project/control-plane/blob/d089a040db5435cb9e53585b26a6188e68dfe5d2/docs/kyma-environment-broker/03-01-service-description.md

Reply:
fixed